### PR TITLE
micro:bit 2021 release blog

### LINKED ISSUE #8273: Micro:bit 2021 Release Blog Post
Hi Galen,
Can you get this blog post prepped for release?
Hoping to release tomorrow (Friday 6/25)
All files here - https://1drv.ms/u/s!AmMIW5Hxi0RtgZ8Sz5hMvcR4ND8kdg?e=0wcNBP
Thanks!
Jacqueline

### PR DESCRIPTION
**Pull request by ganicke**
Import, edit, size images and otherwise prepare the blog post for "micro:bit 2021 Release".

Closes #8273

### CODE REVIEW

File: docs/blog/microbit/2021-release.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+![No text editing message](/static/blog/microbit/2021-release/no-text-editing.png)
+
+If you do need an app with full capabilities but for offline use, we recommend using the [Offline App](https://makecode.microbit.org/offline).
+
+[![MakeCode App logo](/static/blog/microbit/2021-release/app-logo.png)](https://makecode.microbit.org/offline)
+
+Thank you to everyone who has logged bugs, helped with translations, and suggested new features throughout the year! Any new bugs you find in this release, please log them in [GitHub](https://github.com/Microsoft/pxt-microbit/issues).
+
+Any other comments, suggestions, and feedback, please participate in the micro:bit community on [Slack](https://tech.microbit.org/get-involved/where-to-find/). And, please let us know what you think of this year’s updates on Twitter [@MSMakeCode](https://twitter.com/MSMakeCode).
```
Comment (ganicke) [inline docs/blog/microbit/2021-release.md:79]: This Slack link appears to be bad. Did you want https://tech.microbit.org/community/?

File: docs/blog/microbit/2021-release.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+### Every time interval block
+
+This new block is useful for looping on a given time interval (in milliseconds). It's like the Forever loop but the code inside doesn't run again until an interval of time set for the loop expires first.
+
+![New every loop block](/static/blog/microbit/2021-release/every.png)
+
+## Text coding not available in Windows app
+
+Now that one-click downloads are supported with the web version of MakeCode, this release starts the journey to deprecate the Windows 10 app. You can no longer use the text editor in the app, and we will stop updating the app by the 2022 release next June.
```
Comment (ganicke) [inline docs/blog/microbit/2021-release.md:69]:
```suggestion
Now that one-click downloads are supported with the web version of MakeCode, this release starts the journey to deprecate the Windows 10 app. You can no longer use the text editor in the app, and update support for the app will stop after the 2022 release next June.
```

File: docs/blog/microbit/2021-release.md (hunk 1 above)
Comment (Jaqster) [reply docs/blog/microbit/2021-release.md:79]: Ah, yes please!  Thanks for catching that!

**Review by Jaqster — APPROVED** [review]
Looks great!  Just made a couple tweaks.  Thanks Galen!

**Comment by Jaqster** [comment]
Adding @abchatra who will make the call on Monday whether we release...

**Comment by abchatra** [comment]
We decided to wait atleast a week before shipping new CODAL & bluethooth bootloader. Will update next ewek.

**Comment by abchatra** [comment]
@eanders-MS @riknoll

**Comment by ganicke** [comment]
@anzhou5 - I read your email from Friday, briefly, but my agency botched my account renewal for FY22 and my laptop logged me out just moments later. I can't reply quite yet over email so maybe post an update here on what you want to do for the tutorial section image. Thanks.

**Comment by anzhou5** [comment]
@ganicke - thanks so much for being patient with me. Looks like we can go ahead and add back in the blurb about tutorials being released as they will be ready to go in time. I've attached the release image file on this comment as well!
release-img

**Pull request merged**